### PR TITLE
fix: search params - storage terminals endpoint

### PR DESCRIPTION
**Pull request by OllyDS**
They should be using the keyword `term` rather than `name`

Resolves: https://github.com/VorTECHsa/app-issues/issues/1871

### CODE REVIEW

File: vortexasdk/endpoints/storage_terminals.py [hunk 1]
```diff
@@ -61,24 +61,24 @@ def load_all(self) -> StorageTerminalResult:
...
-        name: Union[str, List[str]] = None,
+        term: Union[str, List[str]] = None,
     ) -> StorageTerminalResult:
         """
-        Find all storage terminals matching given type.
+        Find all storage terminals matching given term.
 
         # Arguments
-            type: The type of storage terminal we're filtering on.
+            term: The term of storage terminal we're filtering on.
```
Comment (Selerski) [inline vortexasdk/endpoints/storage_terminals.py:70]: Shouldn't ids be added to this description?

File: .all-contributorsrc [hunk 2]
```diff
@@ -147,6 +147,15 @@
       "contributions": [
         "code"
       ]
+    },
```
Comment (Selerski) [inline .all-contributorsrc:150]: Nice ;)

**Review by Selerski — CHANGES_REQUESTED** [review]

**Review by Selerski — APPROVED** [review]
if tests pass I'm happy to get this in

File: docs/examples/try_me_out/availability_search.ipynb [hunk 3]
```diff
@@ -26,7 +26,18 @@
     },
     "tags": []
    },
-   "outputs": [],
+   "outputs": [
```
Comment (Selerski) [inline docs/examples/try_me_out/availability_search.ipynb:29]: Please clear these - this is going to put your file structure in the public domain :)

**Review by Selerski — CHANGES_REQUESTED** [review]

File: docs/examples/try_me_out/availability_search.ipynb [hunk 4]
```diff
@@ -94,7 +127,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3.8.11 ('venv': venv)",
```
Comment (Selerski) [inline docs/examples/try_me_out/availability_search.ipynb:130]: why does this need to change? same for below like with interpreter and hash

Comment (OllyDS) [reply docs/examples/try_me_out/availability_search.ipynb:130]: These must have been auto-generated as I didn't change these. Will see how to revert

**Review by Selerski — APPROVED** [review]

**Pull request merged**